pkg/ebpf/tracee.go

```diff
@@ -138,6 +138,62 @@ func (t *Tracee) Stats() *metrics.Stats {
 	return t.stats
 }
 
+// initializeBPFEventsStatsMap initializes the events_stats BPF map with zero values
+// and sets it for metrics collection
+func (t *Tracee) initializeBPFEventsStatsMap() error {
+	evtsCountsBPFMap, err := t.bpfModule.GetMap("events_stats")
```

Comment:
```suggestion
	evtsStatsBPFMap, err := t.bpfModule.GetMap("events_stats")
```

nit: to match the map name itself